spacy/attrs.pyx

```diff
@@ -1,3 +1,5 @@
+from .tokens.token import Token
```

Comment:
```suggestion
from .tokens import Token
```

Reply:
Ah, hmm. That is an issue. Why don't we move the `iob_strings` tuple definition from `spacy.tokens` to `spacy.attrs` as a global like `IOB_STRINGS`?

Reply:
I agree, it creates an circular import. It's a good idea, I can do it.

Reply:
Sorry, that wasn't quite what I meant. Leave `Token.iob_strings` as a method (and continue using it in `Doc` and other places), but just define `IOB_STRINGS` in `attrs` as the tuple (we would use a global-looking all caps name for this) and import that in `Token` to return from `Token.iob_strings`.

Reply:
> Sorry, that wasn't quite what I meant. Leave `Token.iob_strings` as a method (and continue using it in `Doc` and other places), but just define `IOB_STRINGS` in `attrs` as the tuple (we would use a global-looking all caps name for this) and import that in `Token` to return from `Token.iob_strings`.

OK, I applied these comment and it looks like as you asked now.